### PR TITLE
0.13.0 build 0

### PR DESCRIPTION
**Pull request by j-wags**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by j-wags** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -52,11 +52,11 @@ outputs:
...
-        - python >=3.8
+        - python >=3.9
       run:
-        - python >=3.8
+        - python >=3.9
         - openmm >=7.6
-        - openff-interchange =0.2
+        - openff-interchange >=0.3.0
         - rdkit
         - ambertools >=20
```
Comment (mattwthompson) [inline recipe/meta.yaml:61]:
```suggestion
        - ambertools =22
```
We might not want to let 23 slip in if we haven't tested it

Comment (j-wags) [reply recipe/meta.yaml:61]: Good point, but I'd rather be permissive now and pin later with a new build if a problem arises. Our deployment tests (conda CI) should let us know quick if this is a problem.

Comment (mattwthompson) [reply recipe/meta.yaml:61]: Can we a least constrain to 22 or newer? 20 allows ParmEd 4 to be brought in, which breaks examples

Comment (j-wags) [reply recipe/meta.yaml:61]: Yes, I'll get that into the next build/release

**Pull request merged**